vendor/knative.dev/eventing/pkg/scheduler/statefulset/autoscaler.go

```diff
@@ -133,10 +133,10 @@ func (a *autoscaler) Start(ctx context.Context) {
 		case <-ctx.Done():
 			return
 		case <-time.After(a.refreshPeriod):
-			a.logger.Infow("Triggering scale down")
+			a.logger.Infow("Triggering scale down", zap.Bool("isLeader", a.isLeader.Load()))
```

Comment:
that's good extra log. Especially that you mentioned it might be that two pods are not leader... now we have _explicit_ log for that.

noce